app/controllers/memberships/change_zusatzsektion_to_families_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2025, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module Memberships
+  class ChangeZusatzsektionToFamiliesController < ApplicationController
+    before_action :authorize, :role, :person, :group
```

Comment:
Wozu werden role, person und group in der before_action geladen? Es reicht doch, diese Methoden nach Bedarf aufzurufen?